### ISSUE TITLE
logs folder is missing

### ISSUE THREAD

**Issue by Iddoyadlin** [tavorperry/semantle-he-cracker#1]
getting the following when running:

```
PS C:\Users\Iddo\Documents\semantle-he-cracker> node .\main.js
14:27:56.108 INFO  ############################## Starting! ##################
14:27:56.109 INFO  Loading all the words from txt file into an array...
node:events:515
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open 'C:\Users\Iddo\Documents\semantle-he-cracker\logs\2022-06-21T11:27:56.log'
Emitted 'error' event on WriteStream instance at:
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\Users\\Iddo\\Documents\\semantle-he-cracker\\logs\\2022-06-21T11:27:56.log'
}

Node.js v18.4.0
```

**Comment by tavorperry** [comment]
Thanks Ido!
I will update the repo. 
For now, please create manually the folders and it should work.

**Comment by tavorperry** [comment]
Fixed on #2
